R/multitables.R

```diff
@@ -104,7 +105,33 @@ newMultitable <- function (formula, data, name, ...) {
         template=lapply(template$dimensions,
             function (x) list(query=x, variable=findVariableReferences(x)))
     )
-    ## TODO: remove "variable" from that--no longer required?
+
     u <- crPOST(shojiURL(data, "catalogs", "multitables"), body=toJSON(payload))
     invisible(Multitable(crGET(u)))
 }
+
+#' Import a Multitable
+#'
+#' Allows you to import a multitable from a different dataset
+#' @param data an object of class \code{CrunchDataset} in which to create the
+#' multitable
+#' @param multitable an object of class \code{Multitable} that you want copied
+#' to the new dataset
+#' @param name character name to give the new multitable object. If omitted,
+#' a default name will be derived from \code{formula}.
+#' @param ... Additional multitable attributes to set. Options include
+#' \code{is_public}.
+#' @return An object of class \code{Multitable}
+#' @examples
+#' \dontrun{
```

Comment:
Start the example like the `newMultitable` one, then do the copy

m <- newMultitable(~ gender + age4 + marstat, data=ds)
copied_m <- importMultitable(another_ds, m)
name(copied_m) # [1] "gender + age4 + marstat"